pkg/cmd/info/cmd.go

```diff
@@ -0,0 +1,100 @@
+package info
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"kitops/pkg/cmd/options"
+	"kitops/pkg/lib/constants"
+	"kitops/pkg/lib/repo"
+	"kitops/pkg/output"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"oras.land/oras-go/v2/errdef"
+	"oras.land/oras-go/v2/registry"
+)
+
+const (
+	shortDesc = `Show the configuration for a ModelKit`
```

Comment:
On the other commands we have used ModelKit as modelkit. We should either change this or the other rest of the docs

Reply:
Yeah, I wasn't clear what our term was, since the readme is using `ModelKit`.

Reply:
Switched to `modelkit` for now, to match the other commands.